pkg/security/probe/model.go

```diff
@@ -45,16 +45,17 @@ func (ev *Event) GetPathResolutionError() error {
 	return ev.pathResolutionError
 }
 
-// ResolveFileInode resolves the inode to a full path
-func (ev *Event) ResolveFileInode(f *model.FileEvent) string {
+// ResolveFilePath resolves the inode to a full path
+func (ev *Event) ResolveFilePath(f *model.FileEvent) string {
 	if len(f.PathnameStr) == 0 {
-		path, err := ev.resolvers.resolveInode(&f.FileFields)
+		path, err := ev.resolvers.resolveFileFieldsPath(&f.FileFields)
 		if err != nil {
 			if _, ok := err.(ErrTruncatedSegment); ok {
 				ev.SetPathResolutionError(err)
 			} else if _, ok := err.(ErrTruncatedParents); ok {
 				ev.SetPathResolutionError(err)
 			}
+			f.PathResolutionError = err
```

Comment:
If we export resolution errors that are not `ErrTruncatedParents` or `ErrTruncatedSegment`, we might need to change the `NewAbnormalPathEvent` in `custom_events.go`